blockage/plugins.py

```diff
@@ -2,6 +2,22 @@
     import httplib
 except ImportError:
     import http.client as httplib
+
+try:
+    unicode = unicode
+except NameError:
+    # 'unicode' is undefined, must be Python 3
+    str = str
+    unicode = str
+    bytes = bytes
+    basestring = (str, bytes)
+else:
+    # 'unicode' exists, must be Python 2
+    str = str
+    unicode = unicode
+    bytes = str
+    basestring = basestring
```

Comment:
:smile: The only thing we're using here is basestring, you can keep that and remove the rest. But I like to have the comparison "py3 vs py2" types, it's better to understand what's happening IMHO